src/components/org-users/views.tsx

```diff
@@ -291,72 +282,76 @@ export function PermissionTable(
           </p>
         </div>
       </details>
-      <table className="govuk-table permissions">
-        <thead className="govuk-table__head">
-          <tr className="govuk-table__row">
-            <th className="govuk-table__header name" scope="col">
+
+      {props.spaces.map((space, index) => (
+        <>
```

Comment:
```suggestion
        <Fragment key={index}>
```

Will fix your warnings.